pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,222 @@
+package mysql
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+	"go.expect.digital/translate/pkg/tracer"
+)
+
+var mysqlRepo *Repo
+
+func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	viper.SetEnvPrefix("translate_mysql")
+	viper.AutomaticEnv()
+
+	tp, err := tracer.TracerProvider()
```

Comment:
otel.SetTracerProvider(tp) to set tracer provider globally?

Reply:
it is already :)